charactersheet/charactersheet/models/spell_slot.js

```diff
@@ -63,3 +66,9 @@ Slot.findAllBy = function(characterId) {
         return e.characterId() === characterId;
     });
 };
+
+Slot.REST_TYPE = {
```

Comment:
If the constants are, or will be, used more than once, I think they should. Up to you.